wavefront/server/apps/call_processing/call_processing/services/pipecat_service.py

```
@@ -574,13 +677,39 @@ async def on_function_calls_started(service, function_calls):
+                outcome = 'unknown'
+            # Pull language switch count from language_state (already tracked there)
+            call_stats['language_switch_count'] = language_state.get('switch_count', 0)
+            asyncio.create_task(
+                CallEvaluationService.record_call_metrics(
+                    call_id=call_id,
+                    agent_config=agent_config,
+                    call_outcome=outcome,
+                    transcript_log=transcript_log,
+                    stats=call_stats,
+                )
+            )
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)